content/reference-transaction-format.md

```diff
@@ -576,9 +577,22 @@ A trust line indicates an issuer you trust enough to accept their issuances as p
+
+When offers of the same exchange rate are placed in the same ledger version, the order in which they are taken is determined by the [canonical order](https://github.com/ripple/rippled/blob/release/src/ripple/app/misc/CanonicalTXSet.cpp "Source: Transaction ordering") in which the transactions were [applied to the ledger](https://github.com/ripple/rippled/blob/5425a90f160711e46b2c1f1c93d68e5941e4bfb6/src/ripple/app/consensus/LedgerConsensus.cpp#L1435-L1538 "Source: Applying transactions"). This behavior is designed to be deterministic, efficient, and hard to game.
+
+#### TickSize
+
+_Requires the [TickSize amendment](concept-amendments.html#ticksize)._
+
+When an Offer is placed into an order book, its exchange rate is truncated based on the `TickSize` values set by the issuers of the currencies involved in the Offer. When a trader offers to exchange XRP and an issued currency, the `TickSize` from the issuer of the currency applies. When a trader offers to exchange two issued currencies, the offer uses the smaller `TickSize` value (that is, the one with fewer significant digits). If neither currency has a `TickSize` set, the default behavior applies.
+
+The `TickSize` value truncates the number of _significant digits_ in the exchange rate of an offer when it gets placed in an order book. Issuers can set `TickSize` to an integer from `3` to `15` using an [AccountSet transaction](#accountset). The exchange rate is represented as a number of significant digits plus an exponent; the `TickSize` does not affect the exponent. This allows the Ripple Consensus Ledger to represent exchange rates between assets that vary greatly in value (for example, a hyperinflated currency compared to a rare commodity). The lower the `TickSize` an issuer sets, the larger the increment traders must offer to be considered a higher exchange rate than the existing Offers.
+
+The `TickSize` does not affect the portion of an Offer that can be executed immediately. (For that reason, OfferCreate transactions with `tfImmediateOrCancel` are unaffected by `TickSize` values.) If the Offer cannot be fully executed, the transaction processing engine calculates the exchange rate and truncates it based on `TickSize`. Then, the engine rounds the remaining amount of the Offer from the "less important" side to match the truncated exchange rate. For a default OfferCreate transaction (a "buy" Offer), the `TakerGets` amount (the amount being bought) gets rounded. If the `tfSell` flag is enabled (a "sell" Offer) the `TakerPays` amount (the amount being sold) gets rounded.
+
```

Comment:
This is partly backwards. The "TakerGets" is the amount being sold, since it's what the person who placed the offer gives to the taker. For a buy, the amount sold is rounded to preserve the amount bought. For a sell, the amount bought is rounded to preserve the amount sold.

To fix the error, just swap "TakerPays" with "TakerGets" in line 592.
For example, suppose there's an offer with a TakerGets of 1 bitcoin and a TakerPays of 1,001 USD. This can be either an offer to buy 1 bitcoin for 1,001 USD (or less) or an offer to sell 1,001 USD for 1 bitcoin (or more). TickSize does not change this, if a but, it still buys 1 bitcoin. If a sell, it still sells 1,001 USD.